client-libraries/0.1/how-to-guides/python/generate-an-address.md

```diff
@@ -0,0 +1,94 @@
+# Generate an address in Python
+
+**In this guide, you learn how to generate a new address for a [seed](root://getting-started/0.1/clients/seeds.md) with a given [security level](root://getting-started/0.1/clients/security-levels.md).**
+
+## Packages
+
+To complete this guide, you need to install the following package (if you're using Go modules, you just need to reference this package):
+
+```bash
+pip install pyota
+```
+
+## IOTA network
+
+In this guide, we connect to a [node](root://getting-started/0.1/network/nodes.md) on the [Devnet](root://getting-started/0.1/network/iota-networks.md#devnet).
+
+## Code walkthrough
+
+1. Import the packages
+
+    ```python
+    from iota import Iota
+    ```
+
+2. Define a seed for which to generate an address
+
+    ```python
+    seed = 'PUETPSEITFEVEWCWBTSIZM9NKRGJEIMXTULBACGFRQK9IMGICLBKW9TTEVSDQMGWKBXPVCBMMCXWMNPDX'
+    ```
+
+3. Connect to a node
+
+    ```python
+    api = Iota('https://nodes.devnet.iota.org:443', seed, testnet = True)
+    ```
+
+    :::info:
+    The `testnet` argument sets the [minimum weight magnitude](root://getting-started/0.1/network/minimum-weight-magnitude.md) to 9.
+    :::
+
+4. Define the security level that you want to use for your address
```

Comment:
maybe a note saying that 2 is the default security level in the lib if one doesn't set anything